README.md

```diff
@@ -1,23 +1,47 @@
-# Republication Tracker Tool
+# Republication Tracker Tool #
+**Contributors:** innlabs  
+**Donate link:** https://inn.org/donate  
+**Tags:** publishers, news  
+**Requires at least:** 4.4  
+**Requires PHP:** 5.3  
+**Tested up to:** 5.2.2  
+**Stable tag:** 1.0.2  
+**License:** GPLv2 or later  
+**License URI:** http://www.gnu.org/licenses/gpl-2.0.html  
 
-A plugin that allows users to add a widget to allow readers to easily acquire Creative-Commons-licensed HTML of articles to facilitate embedding posts on external sites. Includes a tracking mechanism similar to ProPublica's PixelPing.
+Adds a widget to allow readers to easily acquire Creative-Commons-licensed HTML of articles to facilitate embedding posts on external sites. Includes a tracking mechanism similar to ProPublica's PixelPing.
 
-**Requires at least:** 4.1  
-**Tested up to:** 5.2.2
-**Stable tag:** See https://github.com/Automattic/republication-tracker-tool/releases
-**License:** GPLv2 or later  
-**License URI:** https://www.gnu.org/licenses/gpl-2.0.html
+## Description ##
+
+A plugin that allows users to add a widget to allow readers to easily acquire Creative-Commons-licensed HTML of articles to facilitate embedding posts on external sites. Includes a tracking mechanism similar to ProPublica's PixelPing. Built and maintained by [INN Labs](https://labs.inn.org/).
+
+## Installation ##
+
+1. Upload `plugin-name.php` to the `/wp-content/plugins/` directory
+2. Activate the plugin through the 'Plugins' menu in WordPress
+3. Add the widget to your per-post sidebars. It doesn't work outside of single post pages.
+
+## Frequently Asked Questions ##
```

Comment:
Should this section include the other topics from the GitHub `docs/` folder?

Reply:
No strong convictions here. It may.